includes/class-wc-frontend-scripts.php

```diff
@@ -368,7 +368,11 @@ public static function load_scripts() {
 			self::enqueue_style( 'select2' );
 
 			// Password strength meter. Load in checkout, account login and edit account page.
-			if ( ( 'no' === get_option( 'woocommerce_registration_generate_password' ) && ! is_user_logged_in() ) || is_edit_account_page() || is_lost_password_page() ) {
+			if ( is_edit_account_page() ||
+				is_lost_password_page() ||
+				'yes' === get_option( 'woocommerce_registration_generate_password_allow_overwrite' ) ||
+				( 'no' === get_option( 'woocommerce_registration_generate_password' ) && ! is_user_logged_in() )
```

Comment:
I think this might need to be
```php
( ( 'no' === get_option( 'woocommerce_registration_generate_password' ) || 'yes' === get_option( 'woocommerce_registration_generate_password_allow_overwrite' ) ) && ! is_user_logged_in() )
```
As the password meter does not need to be loaded when overwrite is enabled and the user is already logged in.